packages/openactive-integration-tests/test/features/core/amending-order-quote/implemented/amend-c1-and-c2-test.js

```diff
@@ -0,0 +1,118 @@
+const chai = require('chai');
+const chakram = require('chakram');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { B, C1, Common, GetMatch } = require('../../../../shared-behaviours');
+const { FlowHelper } = require('../../../../helpers/flow-helper');
+const { RequestState } = require('../../../../helpers/request-state');
+const { generateUuid } = require('../../../../helpers/generate-uuid');
+const { C2 } = require('../../../../shared-behaviours/c2');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'core',
+  testFeature: 'amending-order-quote',
+  testFeatureImplemented: true,
+  testIdentifier: 'amend-c1-and-c2',
+  testName: 'Amend, at C1 and C2, an existing OrderQuote',
+  testDescription: 'Run C1,C2 with opportunity A, then - with the same Order UUID - run C1,C2 with opportunity B, then runs B. The order should be confirmed for opportunity B',
```

Comment:
"confirmed" doesn't have a meaning in the Open Booking API in the context of the Order, only the OrderItem

```suggestion
  testDescription: 'Run C1,C2 with opportunity A, then - with the same Order UUID - run C1,C2 with opportunity B, then runs B. The resulting Order should include a booking for only opportunity B',
```